cmd/neofs-cli/modules/control/evacuate_shard.go

```diff
@@ -0,0 +1,56 @@
+package control
+
+import (
+	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	"github.com/spf13/cobra"
+)
+
+var evacuateShardCmd = &cobra.Command{
+	Use:   "evacuate",
+	Short: "Evacuate objects from shard",
+	Long:  "Evacuate objects from shard to other shards",
+	Run:   evacuateShard,
+}
+
+func evacuateShard(cmd *cobra.Command, _ []string) {
+	pk := key.Get(cmd)
+
+	req := &control.EvacuateShardRequest{Body: new(control.EvacuateShardRequest_Body)}
+	req.Body.Shard_ID = getShardID(cmd)
+	req.Body.IgnoreErrors, _ = cmd.Flags().GetBool(dumpIgnoreErrorsFlag)
+
+	signRequest(cmd, pk, req)
+
+	cli := getClient(cmd, pk)
+
+	var resp *control.EvacuateShardResponse
+	var err error
+	err = cli.ExecRaw(func(client *client.Client) error {
+		resp, err = control.EvacuateShard(client, req)
+		return err
+	})
+	if resp != nil {
```

Comment:
Do we need this `if`? I guess exactly one non-nil error is returned, so we can just check `common.ExitOnErr` before.

Reply:
Fixed